js/client.js

```diff
@@ -26,9 +26,13 @@
 		}
 	});
 	$(window).on('dragover', function (e) {
+		if (/^text/.test(e.target.type)) // Ignore text fields
```

Comment:
Well, the type of a `<textarea>` is just `textarea`, and I don't think anything else has a type beginning with `text`...